concourse/scripts/pipelines-cloudfoundry.sh

```diff
@@ -11,9 +11,11 @@ config_autodelete="${SCRIPT_DIR}/../pipelines/autodelete-cloudfoundry.yml"
 
 generate_vars_file() {
    set -u # Treat unset variables as an error when substituting
+   local aws_account=${AWS_ACCOUNT:-dev}
    cat <<EOF
 ---
-aws_account: ${AWS_ACCOUNT:-dev}
+aws_account: ${aws_account}
+auto_deploy: $([ "${aws_account}" != "dev" ] && echo "true" || echo "false")
```

Comment:
I'm unsure about us keying feature against environment names. It reminds me of this from 12factor.net:

> Another aspect of config management is grouping. Sometimes apps batch config into named groups (often called “environments”) named after specific deploys, such as the development, test, and production environments in Rails. This method does not scale cleanly: as more deploys of the app are created, new environment names are necessary, such as staging or qa. As the project grows further, developers may add their own special environments like joes-staging, resulting in a combinatorial explosion of config which makes managing deploys of the app very brittle.

Maybe we should default this to the development case and have a named environment variable to enable it? Like we do for `DISABLE_AUTODELETE`.